earth_enterprise/rpms/build.gradle

```diff
@@ -59,8 +62,19 @@ if (new File('build.properties').exists()) {
     }
 }
 
+def openGeeVersionFile = new File("../src/NATIVE-REL-x86_64/gee_long_version.txt")
+
+if (!openGeeVersionFile.exists()) {
+    GeeCommandLine.expand(
+            ["/usr/bin/scons", "release=1", "NATIVE-REL-x86_64/gee_long_version.txt"],
```

Comment:
Is there any way to make this more generic, i.e. by not specifying "release=1" and "NATIVE-REL-x86_64"?  Or is that not necessary since it will only build a single file?

Reply:
If we modify the `scons` task to output the file to generic location, we could use that.  Otherwise, where the file ends up, depends on what `release` flag you pass.  So, you need some way to decide what that's going to be.

In general, RPM packaging uses only release builds.  You can try to trick it to use other builds, but you'll have to make some changes by hand.

This command only executes the targets necessary to produce that single file.